dao/courses.go

```diff
@@ -137,6 +157,14 @@ func GetCourseById(ctx context.Context, id uint) (course model.Course, err error
 	return foundCourse, dbErr
 }
 
+func GetNonHiddenCourseById(ctx context.Context, id uint) (course model.Course, err error) {
+	var foundCourse model.Course
+	dbErr := DB.Preload("Streams.Stats").Preload("Streams.Files").Preload("Streams", func(db *gorm.DB) *gorm.DB {
```

Comment:
I think this doesn't need to preload the stats, as the function is not used in the admin tab.

Reply:
BTW, where is it used?

Reply:
Nowhere actually.😅 Interesting that the test didn't fail for the unused function.

Reply:
It's exported (capitalized), so the linter assumes we might want other projects to import it :smile: